### PR TITLE
CT-817 test datapreview on typed table with specified datatype

### PR DESCRIPTION
**Pull request by jirkasemmler**
Jira: CT-817
KBC: https://github.com/keboola/connection/pull/4262

Before asking for review make sure that:

## Checklist

- [x] New client method(s) has tests
- [x] Apiary file is updated

## Release

  - [x] I gave the PR a proper label:
    * major (BC break)
    * minor (new feature)
    * patch (backwards compatible fix)
    * no release (just test changes)

### CODE REVIEW

File: tests/Backend/Snowflake/TableDefinitionOperationsTest.php [hunk 1]
```diff
@@ -810,4 +810,126 @@ public function testCreateSnapshotOnTypedTable(): void
             ]);
         }
     }
+
+
+    public function testDataPreviewOnTypedTableWithWhereFilters(): void
+    {
+        // TODO - dopsat tam jeste dalsi kombinace typu, staci do tabulky
```
Comment (romanbracinik) [inline tests/Backend/Snowflake/TableDefinitionOperationsTest.php:817]: Dopisal by som tam teda tie dalsie kombinacie  :)

Comment (jirkasemmler) [reply tests/Backend/Snowflake/TableDefinitionOperationsTest.php]: hotovo

**Review by romanbracinik — APPROVED** [review]
🙆‍♂️

**Comment by jirkasemmler** [comment]
@romanbracinik pridal jsem test na export + jsem od-excludoval testy v part-1 suitach, ktere imho tam byly historicky zapomenute

**Review by romanbracinik — APPROVED** [review]
👌

**Pull request merged**